File: app/src/main/java/com/doapps/habits/services/NotificationService.java
package com.doapps.habits.services;

import android.app.Service;
import android.content.Intent;
import android.os.IBinder;

import com.doapps.habits.receivers.Alarm;

public class NotificationService extends Service {
    private final Alarm alarm = new Alarm();

    @Override
    public int onStartCommand(Intent intent, int flags, int startId) {
        alarm.setAlarm(this);
        return START_STICKY;
    }

    @Override
    public IBinder onBind(Intent intent) {
        return null;
    }
}

File: app/src/main/java/com/doapps/habits/services/NotificationService.kt
package com.doapps.habits.services

import android.app.Service
import android.content.Intent
import android.os.IBinder

import com.doapps.habits.receivers.Alarm

class NotificationService : Service() {
    private val alarm = Alarm()

    override fun onStartCommand(intent: Intent, flags: Int, startId: Int): Int {
        alarm.setAlarm(this)
        return Service.START_STICKY
    }

    override fun onBind(intent: Intent): IBinder? {
        return null
    }
}